metric/blockgroup/rCOR.csv

```diff
@@ -1,154 +1,154 @@
-370630021002,14,15,13,17,20,10,12,13
-370630021003,13,14,11,16,14,11,17,14
-370630022001,20,32,19,16,34,27,21,32
-370630023001,13,11,31,51,55,22,16,19
-370630023002,1,0,20,2,20,12,17,1
-370639801001,0,1,1,0,0,0,0,0
+id,y_2012,y_2013,y_2014,y_2015,y_2016,y_2017,y_2018,y_2019,y_2020
+370630001011,5,6,12,15,17,14,19,20,19
+370630001012,3,9,5,12,7,5,10,9,7
+370630001021,3,7,6,13,11,16,11,21,13
+370630001022,4,17,6,7,7,13,14,13,12
+370630002001,4,5,3,5,5,8,13,9,8
+370630002002,14,8,8,13,12,18,18,19,12
+370630002003,11,14,18,15,21,17,17,22,26
+370630003011,5,2,4,7,7,15,14,7,14
+370630003012,3,6,7,9,5,6,8,3,13
+370630003013,7,8,3,10,12,15,12,5,3
+370630003021,17,11,12,20,14,11,18,23,17
+370630003022,7,7,8,20,11,15,20,11,14
+370630003023,13,23,16,19,21,20,19,10,25
+370630004011,5,9,18,14,23,17,16,18,13
+370630004012,18,26,22,27,22,14,41,25,21
+370630004013,14,5,10,21,11,4,12,10,8
+370630004021,13,23,42,43,37,22,31,25,19
+370630005001,21,30,12,7,9,1,1,0,27
+370630005002,7,5,13,8,11,3,5,5,10
+370630005003,16,7,13,12,11,10,7,7,15
+370630005004,9,15,10,14,10,11,16,9,21
+370630006001,36,17,19,19,28,22,28,37,25
+370630006002,11,9,17,20,19,13,15,20,15
+370630006003,7,8,12,16,18,12,10,18,8
+370630007001,5,7,9,11,12,4,6,8,8
+370630007002,20,19,28,26,43,15,20,13,25
+370630007003,10,10,12,7,16,9,11,8,16
+370630009001,2,4,4,5,4,6,12,10,5
+370630009002,1,10,12,12,20,30,36,31,17
+370630010011,10,4,3,6,7,6,9,20,13
+370630010012,5,7,4,13,20,15,19,18,15
+370630010013,5,6,16,10,13,26,12,9,23
+370630010021,1,1,3,4,4,4,8,2,4
+370630010022,0,1,1,1,1,2,2,1,3
+370630010023,1,1,0,3,4,4,13,9,8
+370630010024,3,11,3,4,7,5,6,9,6
+370630011001,27,26,31,19,24,28,22,16,11
+370630011002,21,7,2,5,13,14,8,7,12
+370630013011,11,5,7,9,14,16,16,27,23
+370630013031,6,1,3,2,5,8,13,29,7
+370630013032,2,1,1,2,1,3,7,5,2
+370630013041,3,3,4,4,9,4,11,4,15
+370630014001,2,5,0,1,0,1,2,2,1
+370630014002,2,2,1,4,6,1,6,3,4
+370630015011,12,34,0,10,4,1,3,7,3
+370630015021,4,2,1,9,2,5,2,1,1
+370630015022,0,0,0,0,7,3,1,3,0
+370630015023,0,4,4,0,2,0,3,18,18
+370630015024,28,0,1,1,1,0,3,20,11
+370630015031,3,7,2,4,1,3,0,4,2
+370630016011,13,10,10,12,19,13,12,4,10
+370630016012,9,15,11,14,15,10,10,12,16
+370630016013,6,1,4,11,14,5,10,13,7
+370630016031,3,3,3,5,4,4,5,1,5
+370630016032,29,26,46,22,10,10,8,4,12
+370630016033,4,3,6,6,8,9,8,7,12
+370630016034,4,5,5,9,11,10,12,11,16
+370630016041,13,10,8,12,8,18,9,16,15
+370630016042,14,14,18,27,25,21,25,19,18
+370630016043,14,9,5,8,15,16,12,13,10
+370630017051,2,3,10,8,10,14,9,21,11
+370630017052,23,14,15,20,15,19,33,28,13
+370630017053,4,4,4,6,3,6,6,4,4
+370630017061,82,82,121,44,3,10,13,5,14
+370630017062,7,5,2,10,5,10,29,29,58
+370630017071,6,5,7,6,9,12,9,10,8
+370630017072,11,8,18,13,26,20,13,7,17
+370630017073,3,4,5,8,5,3,13,6,7
+370630017074,14,23,17,16,28,19,40,47,16
+370630017075,9,14,34,34,27,31,36,44,74
+370630017081,23,89,5,8,9,6,4,2,3
+370630017091,0,1,1,2,1,0,4,1,3
+370630017092,0,1,0,0,5,4,4,0,1
+370630017093,2,0,3,4,33,39,42,97,95
```

Comment:
A few 2019 values changed?

Reply:
can also verify that this 97 count for 2019 matches up with what is in postgis, again not sure what's going on